src/Components/Booking/Components/Modals/End-Ride/endRideConfirm.component.js

```diff
@@ -97,6 +97,7 @@ export default class EndRideConfirm extends Component {
             if (result.success) {
                 ModalManager.closeModal();
                 ModalManager.closeModal();
+                window.location.reload(true);
```

Comment:
Why do we need to reload entire window here?

Reply:
data refresh and status was not reflecting

Reply:
Use callback method to refresh data. Refreshing entire window spoils user experience and should be avoided .